apps/projection/src/app/ui/card/card.component.html

```diff
@@ -1,29 +1,25 @@
-<div class="border-2 border-black rounded-md p-4 w-fit flex flex-col gap-3" [class]="customClass">
-  <img
-    *ngIf="type === CardType.TEACHER"
-    src="assets/img/teacher.png"
-    width="200px"
-  />
-  <img
-    *ngIf="type === CardType.STUDENT"
-    src="assets/img/student.webp"
-    width="200px"
-  />
-
+<div
+  class="border-2 border-black rounded-md p-4 w-fit flex flex-col gap-3"
+  [class]="customClass">
+  <ng-container
+    [ngTemplateOutlet]="
+      templateView ? templateView : defaultView
+    "></ng-container>
   <section>
     <app-list-item
       *ngFor="let item of list"
-      [name]="item.firstname"
+      [name]="item.firstname ? item.firstname : item.name"
       [id]="item.id"
       [type]="type"
-    >
+      (deleteEvent)="deleteItem($event)">
     </app-list-item>
```

Comment:
look at ng-template-outlet to deal with this part.

Reply:
done